packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -74,6 +136,34 @@ test.describe('PieInput - Component tests', () => {
                 expect((await input.inputValue())).toBe('test');
             });
         });
+
+        test.describe('name', () => {
+            test('should default to an empty string if no name prop provided', async ({ mount }) => {
```

Comment:
Does this actually default to an empty string? I thought that `ifDefined` will not set an attribute if the value does not exist?

Reply:
Actually i think i see now. I think it is removing it but the test description may be a little misleading. Probably needs to be read: `should not render a name attribute if no name prop is provided`?

Reply:
in my testing, even with that approach if you programmatically query an `input` element for the attribute (name and value at least) then it will return an empty string in the browser. So I think Lit is not adding anything, and browser defaults to empty string but not as an attribute you can see in the inspected html